ads.txt

```diff
@@ -28,7 +28,7 @@ google.com, pub-2012933198307164, DIRECT
 google.com, ca-pub-7526622147976638, DIRECT
 google.com, ca-pub-4979300716531995, DIRECT
 google.com, pub-5107472231497753, DIRECT
-go.sonobi.com, d089631d2d, DIRECT
+go.sonobi.com, d089631d2d, RESELLER
```

Comment:
I think we should double check this with Danny and Drew, just in case there's a reason they've put them as DIRECT originally.

Reply:
Confirmed this needs to be direct, can you change back?

![image](https://user-images.githubusercontent.com/11922253/29870880-82d23cee-8d81-11e7-87aa-50f9013be6e4.png)